src/app/pages/sharing/nvme-of/nvme-of.store.ts

```diff
@@ -0,0 +1,86 @@
+import { computed, Injectable } from '@angular/core';
+import { ComponentStore } from '@ngrx/component-store';
+import {
+  combineLatest, switchMap, tap,
+} from 'rxjs';
+import {
+  NvmeOfNamespace, NvmeOfSubsystem, SubsystemHostAssociation, SubsystemPortAssociation,
+} from 'app/interfaces/nvme-of.interface';
+import { ApiService } from 'app/modules/websocket/api.service';
+
+export interface NvmeOfState {
+  subsystems: NvmeOfSubsystem[];
+  namespaces: NvmeOfNamespace[];
+  hostSubsystems: SubsystemHostAssociation[];
+  ports: SubsystemPortAssociation[];
+  isLoading: boolean;
+}
+
+const initialState: NvmeOfState = {
+  subsystems: [],
+  namespaces: [],
+  hostSubsystems: [],
+  ports: [],
```

Comment:
Like `hostSubssytems` it's also not a list of all ports, but a list of relations between subsystems and ports.